include/bluetooth/hci.h

```diff
@@ -1335,6 +1335,48 @@ struct bt_hci_cp_le_set_privacy_mode {
 	uint8_t         mode;
 } __packed;
 
+#define BT_HCI_LE_AOA_CTE_RESP                  BIT(0)
+#define BT_HCI_LE_AOD_CTE_RESP_1US              BIT(1)
+#define BT_HCI_LE_AOD_CTE_RESP_2US              BIT(2)
+
+#define BT_HCI_LE_SWITCH_PATTERN_LEN_MIN    0x2
+#define BT_HCI_LE_SWITCH_PATTERN_LEN_MAX    0x4B
+
+#define BT_HCI_OP_LE_SET_CONN_CTE_TX_PARAMS     BT_OP(BT_OGF_LE, 0x0055)
+struct bt_hci_cp_le_set_conn_cte_tx_params {
+	uint16_t handle;
+	uint8_t cte_types;
```

Comment:
Minor, but if you check other structs in this header file which have a mix of uint16_t and uint8_t types, the uint8_t lines have an extra space between the type and the variable name so that they align the same way with uint16_t types, etc.

Reply:
Added white-spaces to line up members.